js/lib/cachetasks.js

```diff
@@ -95,7 +95,17 @@
       case "store":
         try {
           response = yield fetch(request);
-          yield cache.put(request, response.clone());
+
+          // Create response with timestamp appended.
+          if (cacheName == "directory_links") {
```

Comment:
We should keep CacheTasks is as a generic library. If we need any tasks related to DirectoryLinksProvider, they should be part of that object and call into this object.